docs/outputs.rst

```diff
@@ -319,14 +321,74 @@ Ephemeris Output
++------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
+| Obj_Sun_vz_LTC_km_s                | Float        | Heliocentric object-sun light-time-corrected Cartesian z velocity (km/s)                                 |
++------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
+| Obj_Sun_x_km                       | Float        | Heliocentric object-sun Cartesian x distance (km)                                                        |
++------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
+| Obj_Sun_y_km                       | Float        | Heliocentric object-sun Cartesian y distance (km)                                                        |
++------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
+| Obj_Sun_x_km                       | Float        | Heliocentric object-sun Cartesian z distance (km)                                                        |
++------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
+| Obj_Sun_vx_km_s                    | Float        | Heliocentric object-sun Cartesian x velocity (km/s)                                                      |
++------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
+| Obj_Sun_vy_km_s                    | Float        | Heliocentric object-sun Cartesian y velocity (km/s)                                                      |
++------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
+| Obj_Sun_vz_km_s                    | Float        | Heliocentric object-sun Cartesian z velocity (km/s)                                                      |
++------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
+| phase_deg                          | Float        | The sun-object-observer angle (degrees)                                                                  |
++------------------------------------+--------------+----------------------------------------------------------------------------------------------------------+
+
+.. note::
+   All positions, positions, and velocities are in respect to J2000.
+
+.. note::
+   All columns in the comple physicalx parameters file will also be included in the full output.
+
```

Comment:
We have this under the inputs section  - we should directly link there, but I think better to not duplicate here otherwise we have things to maintain  - check the inputs is correct - I am pretty sure it is.